challenges/objects-arrays.js

```diff
@@ -89,32 +113,42 @@ The zoo wants to display both the scientific name and the animal name in front o
 
 */
 const animalNames = [];
-console.log(animalNames);
+let names = zooAnimals.forEach(function(i){
+  zooAnimals.push(i.scientific_name + i.animal_name);
```

Comment:
I don't think that you will want to push this information into the zooAnimals array. Look around and see if there is another array that has been defined that would be a better place to put these names.